src/main/java/sirius/kernel/commons/Files.java

```diff
@@ -22,6 +26,8 @@
  */
 public class Files {
 
+    protected static final Log LOG = Log.get("files");
```

Comment:
no we need a specific logger? this is essentially a "cannot happen" case -> handle via default logger